packages/bootstrap/package.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Telerik.Kendo.ThemeBootstrap</id>
+    <version>$VERSION$</version>
+    <title>TelerikUI Bootstrap Theme</title>
+    <description>Bootstrap theme for TelerikUI for Blazor</description>
+    <authors>Progress</authors>
+    <owners>Progress</owners>
+    <readme>README.md</readme>
+    <projectUrl>https://github.com/telerik/kendo-themes</projectUrl>
+    <license type="expression">Apache-2.0</license>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <copyright>Copyright © 2022 Progress Software Corporation</copyright>
+    <contentFiles>
+      <files include="README.md" buildAction="None" copyToOutput="false" />
+      <files include="CHANGELOG.md" buildAction="None" copyToOutput="false" />
+      <files include="LICENSE.md" buildAction="None" copyToOutput="false" />
+      <files include="dist/*.css" buildAction="Content" copyToOutput="true" flatten="false" />
+      <files include="dist/*.scss" buildAction="None" copyToOutput="false" />
+      <files include="scss/**/*.scss" buildAction="None" copyToOutput="false" />
+    </contentFiles>
+    <dependencies>
+      <dependency id="Telerik.Kendo.ThemeCore" version="[$VERSION$]" />
+      <dependency id="Telerik.Kendo.ThemeUtils" version="[$VERSION$]" />
+      <dependency id="Telerik.Kendo.ThemeDefault" version="[$VERSION$]" />
+      <dependency id="bootstrap.sass" version="[5.2.1]" />
+    </dependencies>
+  </metadata>
```

Comment:
Wondering if both Core, Utils, Default and Bootstrap are required here - the corresponding [npm](https://www.npmjs.com/package/@progress/kendo-theme-bootstrap) package is dependent on:
- [@progress/kendo-theme-core](https://www.npmjs.com/package/%40progress%2Fkendo-theme-core) [@progress/kendo-theme-default](https://www.npmjs.com/package/%40progress%2Fkendo-theme-default) [bootstrap](https://www.npmjs.com/package/bootstrap)

The same thing for material depends on Core, Utils and Default, while the [npm](https://www.npmjs.com/package/@progress/kendo-theme-material) package is dependent on:
- [@progress/kendo-theme-core](https://www.npmjs.com/package/%40progress%2Fkendo-theme-core) [@progress/kendo-theme-default](https://www.npmjs.com/package/%40progress%2Fkendo-theme-default)

Should the list of dependencies be the other way around?

Reply:
future compatible tweak